src/index.css

```diff
@@ -187,3 +187,27 @@
 @utility animate-slide-in {
   animation: slideIn 0.5s ease-out;
 }
+
+/* Custom scrollbar styles */
+.scrollbar-thin {
```

Comment:
We probably need to make the main scrollbar the same color and width? I'm not sure if the designer was referring to that.